modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -48,6 +61,19 @@ public PythonClientExperimentalCodegen() {
                 .build();
     }
 
+    @Override
+    public void processOpts() {
+        super.processOpts();
+        modelTemplateFiles.put("python-experimental/model.mustache", ".py");
+        supportingFiles.add(new SupportingFile("python-experimental/model_utils.mustache", packagePath(), "model_utils.py"));
+        apiTemplateFiles.put("python-experimental/api.mustache", ".py");
+        supportingFiles.add(new SupportingFile("python-experimental/api_client.mustache", packagePath(), "api_client.py"));
+
+        // default this to true so the python ModelSimple models will be generated
+        ModelUtils.setGenerateAliasAsModel(true);
```

Comment:
I would suggest adding a LOGGER.info to inform the users this setting is hard-coded to true (in other words, there's no way to switch it off)

Reply:
@wing328 I have added a logger.info if the user passes this argument in telling them 
"generateAliasAsModel is hard coded to true in this generator. Alias models will only be generated if they contain validations or enums"